.github/workflows/build-cpp-windows-packages.yml

```diff
@@ -49,6 +49,26 @@ jobs:
       - name: Setup Java
         uses: ./ice/.github/actions/setup-java
 
+      - name: Build C++ Binaries
+        run: msbuild /m ice.proj /t:BuildDist /p:BuildAllConfigurations=yes
+        working-directory: cpp/msbuild
+
+      - name: Sign C++ Binaries with Trusted Signing
+        uses: azure/trusted-signing-action@v0
+        with:
+          azure-tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          azure-client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          azure-client-secret: ${{ secrets.AZURE_CLIENT_SECRET }}
+          endpoint: https://eus.codesigning.azure.net/
+          trusted-signing-account-name: vscx-codesigning
+          certificate-profile-name: vscx-certificate-profile
+          files-folder: ./ice/cpp/bin
+          files-folder-recurse: true
+          files-folder-filter: exe,dll
+          file-digest: SHA256
+          timestamp-rfc3161: http://timestamp.acs.microsoft.com
+          timestamp-digest: SHA256
+
       - name: Build C++ NuGet Packages
```

Comment:
One thing not clear, if the Pack, would trigger a rebuild here.

Previously our custom sign action took care of adding the signatures as part of the dependencies.